app/main.py

```diff
@@ -1,3 +1,12 @@
 def cache(func):
-    # Write your code here
-    pass
+    cash_storJ = {}
```

Comment:
don't use uppercase letters in variable names and make it more readable for a regular programmer :)